tests/integration_tests/tasks/async_queries_tests.py

```diff
@@ -37,18 +43,37 @@
 )
 from tests.integration_tests.test_app import app
 
+# Define the cache backends once
+cache_backends = {
+    "RedisCacheBackend": mock.Mock(spec=RedisCacheBackend),
+    "RedisSentinelCacheBackend": mock.Mock(spec=RedisSentinelCacheBackend),
+    "redis.Redis": mock.Mock(spec=redis.Redis),
+}
+
```

Comment:
I believe these are now redundant as we have the parametrized tests?
```suggestion
# Define the cache backends once
cache_backends = {
    "RedisCacheBackend": mock.Mock(spec=RedisCacheBackend),
    "RedisSentinelCacheBackend": mock.Mock(spec=RedisSentinelCacheBackend),
    "redis.Redis": mock.Mock(spec=redis.Redis),
}

```

Reply:
Thanks, I'll get this removed.

Reply:
Handled in last commit